lib/definition.js

```diff
@@ -354,18 +343,18 @@ module.exports = class Definition {
 
   _validateList() {
     assert(this._list_type, `Invalid Env Config (${this._name}): Missing .type()`)
-    const is_array = Array.isArray(this._value)
-    if (this._required && (!is_array || !this._value.length)) {
+    if (!Array.isArray(this._value)) return
+
+    if (this._required && !this._value.length) {
       throw new MissingEnvError({
-        type: 'list'
+        type: this._type
       , name: this._env
       })
     }
 
-    if (!is_array) return
-
     const type = this._list_type.charAt(0).toUpperCase() + this._list_type.slice(1)
     const method = `_validate${type}`
+    if (!this[method]) return
```

Comment:
What doesn't have a validation method?

Reply:
string.  The validation got a little redundant and the string validation didn't appear to be doing much except in the list context.  Here, it does `String(value).split`, so you've always got an array of strings, or an empty array.  I removed this function because it was impossible to cover since `_populate` handles the `required` checks